### PR TITLE
Removes cigarettes from paramedics mouths

### PR DESCRIPTION
**Pull request by CinderWC**
## About The Pull Request
This PR removes Cigarettes from Paramedic's mouths (It's petty i know, but it's bothering the hell out of me

## Why It's Good For The Game
It doesn't make sense for Paramedics to spawn with cigarettes in their mouth:
First of all the entire medbay is supposed to not allow smoking by default.
Secondly not everyone likes spawning with a cigarette in their mouth, especially those that are anti-smoking.

## Changelog
:cl:
del: Deleted a line of code that made Paramedics spawn with a cigarette in their mouth
/:cl:

### CODE REVIEW

File: code/modules/jobs/job_types/paramedic.dm [hunk 1]
```diff
@@ -26,9 +26,9 @@
 	ears = /obj/item/radio/headset/headset_med
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
 	uniform = /obj/item/clothing/under/rank/medical/paramedic
-	mask = /obj/item/clothing/mask/cigarette
 	shoes = /obj/item/clothing/shoes/jackboots
 	head = /obj/item/clothing/head/soft/emt
+	mask = null //Skyrat Edit
```
Comment (Trilbyspaceclone) [inline code/modules/jobs/job_types/paramedic.dm:31]: oh god dont use null
Just do //mask = /obj/item/clothing/mask/cigarette - Skyrat Edit
This messes with loadout spawning

Comment (ShadeAware) [inline code/modules/jobs/job_types/paramedic.dm:31]: Comment this out broje, making it null is a bit hacky and is gonna fuck with a lot of things.

Comment (CinderWC) [reply code/modules/jobs/job_types/paramedic.dm:31]: Useroth himself told me to do it like that lmao

Comment (CinderWC) [reply code/modules/jobs/job_types/paramedic.dm:31]: Useroth himself told me to do it like that lmao

Comment (Useroth) [reply code/modules/jobs/job_types/paramedic.dm:31]: I confirm. It wouldn't fuck with things, but I agree that would be hacky.

**Pull request merged**